packages/nouns-contracts/contracts/BostonManager.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.6;
+
+contract BostonManager {
+    uint256 public reservePrice;
+    uint8 public minBidIncrementPercentage;
+    uint256 public duration;
+    uint256 public timeBuffer;
+
+    // Sets reserve price on contract deploy
+    constructor(uint256 _reservePrice, uint8 _minBidIncrementPercentage, uint256 _duration, uint256 _timeBuffer) {
+        reservePrice = _reservePrice;
+        minBidIncrementPercentage = _minBidIncrementPercentage;
+        duration = _duration;
+        timeBuffer = _timeBuffer;
+    }
+
+    // Returns reservePrice
+    function getReservePrice() public view returns (uint256) {
```

Comment:
FYI these getters can be removed:

since reservePrice/minBidIncrementPercentage/duration/timeBuffer are public, these getters are automatically created by solidity. ie with `reservePrice()`,`duration()`, etc.